docs/AppTutorial.md

```diff
@@ -0,0 +1,472 @@
+
+## Audience
+This tutorial is intended for people with all amounts of web development experience. If you want to jump straight to code, you may want to skip this tutorial and go to the [developer guidelines](https://github.com/aregee/design-system/blob/master/CONTRIBUTING.md) page.
+
+## Prerequisites
+* Node JS
+* Git
+
+## Getting Started
+Each step in this guideline illustrates a different aspect of developing web applications with Innovaccer design system. Refer the following sections below to get familiar with this project:
+* [Initializing App](#APP_INIT)
+* [Installing component Library](#COMPONENT_LIBRARY)
+* [Creating page grid](#BUILDING_LANDING_PAGE)
+* [Building Drilled Page](#BUILDING_DRILLED_PAGE)
+
+
+## <a name="APP_INIT"></a>Initializing App
+We will start with `Create React App`.
+```jsx
+npx create-react-app covid
+cd covid
+npm start
+```
+Next, you need to install Innovaccer design system in your project using the following command :
+```bash
+npm install @innovaccer/design-system
+```
+Follow [developer guidelines](https://github.com/aregee/design-system/blob/master/README.md) to install style.
+
+## <a name="COMPONENT_LIBRARY"></a>Installing component Library
+A preview of what you will build:
+
+![alt tag](./images/Covid-App.png)
+Lets begin by creating the basic Header and Footer components. We will be adding *Heading.js* and *Footer.js* in our src folder.
+```jsx
+src
+├──Header.js
+├──Footer.js
+```
+Here is a preview of Header component:
+
+![alt tag](./images/Header.png)
+
+So, we will be needing *Heading* component from our design-system package.
+```jsx
+import { Heading } from 'design-system';
+```
+Heading component can be used in the following way:
+```jsx
+<header>
+    <Heading size="l" appearance="white">
+        Covid-19
+    </Heading>
+</header>
+```
+Now we will create a Footer for our app. Here is a preview of Footer component:
+
+![alt tag](./images/Footer.png)
+
+For the footer, we are going to use *Heading* and *Text* component. In *Footer.js*, add these lines:
+```jsx
+import { Link, Text } from 'design-system';
+```
+We can use Text and Link inside footer element in the following way:
+```jsx
+ <footer className="App-footer">
+    <Text appearance="white" weight="strong">
+        Copyright Innovaccer
+    </Text>
+    <Link onClick={() => history.push('/references')}>References</Link>
+</footer>
+```
+### Add Routing
+We’ve updated our app to render, but now we need to add routing functionality to handle our drilled states. To do this we first need to install react-router-dom.
+```jsx
+npm install --save react-router-dom
+```
+Now, we need to wrap our app in the Router component. In the root *index.js*, add the import:
+```jsx
+import { HashRouter as Router, Switch, Route } from 'react-router-dom';
+```
+Then, update the render() function to include the Router.
+```jsx
+<div>
+    <Header />
+    <Router basename={process.env.PUBLIC_URL}>
```

Comment:
I guess process.env.PUBLIC_URL is for github pages. Replacing it with '/' would be better.